template/template.go

```diff
@@ -64,12 +64,18 @@ func init() {
 
 	HelperFuncs = template.FuncMap{
 		// Misc functions that operate on IfAddrs inputs
-		"attr":   sockaddr.IfAttr,
+		"attr":   sockaddr.IfAttrs,
 		"join":   sockaddr.JoinIfAddrs,
 		"limit":  sockaddr.LimitIfAddrs,
 		"offset": sockaddr.OffsetIfAddrs,
 		"unique": sockaddr.UniqueIfAddrsBy,
 
+		// Misc math functions that operate on a single IfAddr input
+		"math": sockaddr.IfAddrsMath,
+
+		// Misc functions that operate on IfAddr input
+		"Attr": sockaddr.IfAttr,
```

Comment:
Having `Attr` and `attr` seems super confusing - is there a better name we can give the new one?